flocker/node/agents/functional/test_cinder_behaviour.py

```diff
@@ -56,7 +59,7 @@ def test_create_metadata_is_listed(self):
         expected_metadata = {random_name(self): "bar"}
 
         new_volume = self.cinder_volumes.create(
-            size=100,
```

Comment:
:question: Sad panda at `ICinderVolumeManager`. I would have hoped `size` was named `size_gib`.

Either way thanks for fixing this value.